### ISSUE TITLE
Request for a hello-world example

### ISSUE THREAD

**Issue by dov** [vsg-dev/vsgExamples#293]
I'm missing a example that builds a triangle directly from vertices, and turns it into a vsg::Node.

Here are some additional features that I'd like to see/learn from such an example:

1. How to add color per vertex, like in the hello triangle example from the Vulkan Tutorial
2. How to add normals and texture coordinates to the VBO.
3. How to add the mesh both directly and without indices.
4. How to create a VBO with only vertices and do colors and normals in the shaders through uniforms (to save memory!)

**Comment by robertosfield** [comment]
The vsgdraw example is the first VSG example that I wrote, it started like as a reimplementation of the Vulkan Tutorial.

https://github.com/vsg-dev/vsgExamples/tree/master/examples/commands/vsgdraw

VBO doesn't really mean anything in Vulkan, do you mean an OpenGL Vertex Buffer Object?

As general comment, we provide all the software, documentation and support for free, but don't take requests to do further work for free just because we already have given a buckload of our time for free.

**Comment by dov** [comment]
@robertosfield Thanks for the pointer to vsgdraw. That's very near what I was looking for. I still feel however, that creating a simple mesh object from vertices can be made simpler. Have you considered adding a `createMesh()` method to the vsg::Builder class? I was thinking along the lines of something like `SoIndexedFaceSet` in OpenInventor.

Regarding VBO, my mistake. I simply meant Vertex Buffers.

On your general comment, first let me know that I a very much appreciate and am in awe of the fantastic work that you and others have put into the project! 

This issue that I opened should in no way be considered a "demand" of you writing something, and it should in no way take away form the appreciation I feel for the project! The way I treat issues on github, also on "my" repositories, is that a user has a general wish for something to "occur", and perhaps one day, someone else will be inclined to carry out that wish. There are a lot of issues on github that sit for years as open, and most will probably never be closed. Still, as a user, encountering such issues gives you an idea of what people have been cooping with and what features requests are not implemented. So imho, as a maintainer, and original author of the project, you certainly have the freedom to interpret issues different, and I respect that.

So I opened this issue as a sort of a brainstorm of things that I'd like to learn, and reflecting pain points in my learning process.

Keep up the good work!
